blog/tooltip-w-teorii-i-praktyce/index.md

```diff
@@ -0,0 +1,168 @@
+---
+title: 'Tooltips w teorii i praktyce'
+authors: [toporek]
+date: '2024-09-25'
+tags:
+  - 'z kraju'
+  - 'edukacja'
+  - 'dobre-praktyki'
+coverImage: 'tooltips cover.png'
+---
+
+Jeśli chcecie nadać nowy „wygląd” Waszej aplikacji, wprowadzić nową funkcję lub
+ulepszyć istniejącą, tooltipy mogą Wam w tym pomóc. Tooltipy mogą odegrać
+kluczową rolę w poprawie użyteczności aplikacji, zmniejszając dezorientację
+użytkowników i ułatwiając efektywne wykonywanie zadań. W tym artykule
+wyjaśniamy, czym są tooltipy i jak z nich korzystać.
+
+<!--truncate-->
+
+Ten artykuł będzie przewodnikiem krok po kroku, w jaki sposób tooltipy mogą
+pomóc użytkownikom uniknąć rozczarowania i pozwolić im kontynuować naukę i
+wdrażanie funkcji. Na początek może, małe wprowadzenie.
+
+## Czym dokładnie są tooltipy?
+
+[Tooltipy](https://pl.wikipedia.org/wiki/Tooltip) (podpowiedzi), to krótkie
+komunikaty wyświetlane przez użytkownika, które dostarczają dodatkowych
+informacji o wybranym elemencie strony i zachęcają użytkowników do podjęcia
+działań. Tooltipy są zwykle wyświetlane poprzez najechanie myszą lub poprzez
+odpowiednie użycie klawiatury.
+
+## Dlaczego warto ich używać?
+
+Istnieje wiele powodów, dla których warto używać toolipów w aplikacji, oto
+niektóre z nich:
+
+- zajmują niewielką ilość miejsca na ekranie, więc nie są uciążliwą formą
+  przesyłania wiadomości w aplikacji.
+- poprawiają użyteczność i ułatwiają wykonywanie określonych zadań.
+- zapewniają użytkownikom informacje na żądanie w momencie, gdy są potrzebne,
+  dzięki czemu użytkownicy mogą podejmować decyzje, skuteczniej poruszać się po
+  interfejsach i rozumieć funkcjonalność elementów interaktywnych.
+- wyświetlają się tylko kontekstowo, dostarczając dodatkowych wyjaśnień i
+  wskazówek. Swoją drogą, jeśli byłyby wyświetlane na stałe, zaśmiecałyby
+  interfejs użytkownika 😉.
+- mogą być nieocenionym wsparciem dla nowych użytkowników, ułatwiając im
+  zrozumienie funkcji, ich poszczególnych elementów i umożliwiając im interakcję
+  z treściami internetowymi. A jak? Na przykład poprzez przypomnienie, co robi
+  dana ikona lub gdy jakaś funkcja jest trudna do wyjaśnienia są pomocne w
+  nawigowaniu.
+- można je stosować zarówno do prezentacji produktów, wdrażania funkcji, czy też
+  szkolenia użytkowników.
+
+## Gdzie warto je stosować?
+
+Wykorzystanie tooltipów może być dość wszechstronne w różnych biznesach, ale
+skupmy się na kilku typowych w branży technologicznej. Można do nich zaliczyć
+między innymi:
+
+- wycieczki wdrożeniowe – gdy użytkownik po raz pierwszy loguje się do
```

Comment:
ewentualnie mozesz rozbic to na dwa punkty: wycieczki wdrozeniowe i interaktywne przewodniki. 
tak biegly w tym nie jestem, troche mam klopot z rozroznieniem tych dwoch koncepcji jak to czytam.